loops_lab.js

```diff
@@ -0,0 +1,109 @@
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+// let num = 24;
+
+while (num-- && num >= 2) {
```

Comment:
It's not a good practice to put while loop increment inside loop condition as it will give different result than when you put it inside loop code block. 
Correct way to write while loop should be like this: 
while (num >= 2) {
console.log(num)
num--
}